acceptance/testdata/builder/container/cnb/buildpacks/hello_world/0.0.1/bin/build

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+echo "---> Hello World buildpack"
+
+# INPUT ARGUMENTS
+platform_dir=$2
+env_dir=${platform_dir}/env
+layers_dir=$1
+plan_path=$3
+
+# PLATFORM DIR
+echo "     platform_dir files:"
+ls -alR "${platform_dir}" | sed 's/^/       /'
+
+# ENV VARS
+echo "     env_dir: ${env_dir}"
+echo "     env vars:"
+if compgen -G "${env_dir}/*" > /dev/null; then
+  for var in ${env_dir}/*; do
+    declare "$(basename ${var})=$(<${var})"
+  done
+fi
+export | sed 's/^/       /'
+
+# LAYERS
+echo "     layers_dir: ${layers_dir}"
```

Comment:
I wonder if we can make assertions around this output to verify that the builder was setup correctly: https://github.com/buildpacks/lifecycle/blob/771ac0b9d0d56cd3baaf28355498a0a636ea6182/cmd/lifecycle/builder.go#L85-L91

Reply:
Yeah, it sounds good to me. I have some problems with `plan.toml`and `group.toml`. I am using these toml files as empty right now, and builder is not even using buildpack. I need to look some docs and fill these tomls correctly. After that i will coverage some basic test cases ([like `buildpack.toml` api test](https://github.com/buildpacks/lifecycle/pull/648#issuecomment-870823450)). Then i can do these assertions.

Reply:
I will start this one at daytime, i think all other test cases are done for me. We need to cover some success scenarios and some test cases need to check output (like structure, inside files, toml files keys).

Reply:
Done 👍 , please check this commit https://github.com/buildpacks/lifecycle/pull/648/commits/7233dad5192fb8318d0ec4805e74b19296e138e5

Reply:
Looks good!